fuel-tx/src/transaction/validity/error.rs

```diff
@@ -112,6 +116,8 @@ pub enum ValidityError {
     /// The total number of bytecode subsections in the `Upload` transaction exceeds the
     /// limit.
     TransactionUploadTooManyBytecodeSubsections,
+    /// Blob id of the transaction differs from the data.
+    TransactionBlobIdVerificationFailed,
```

Comment:
The same here, the error should be at the end

Reply:
We need a test case for this error=)

Reply:
6fcafd41